src/styles/cards.scss

```diff
@@ -0,0 +1,77 @@
+.cards {
+  background-color: $white-color;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid $gray-color;
+  padding: 32px 16px 16px;
+  &__laptop {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+  &__title {
+    width: 166px;
+    margin-bottom: 4px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $title-color;
+  }
+  &__id {
+    color: $dark-color;
+    font-weight: 400;
+    line-height: 14px;
+    font-size: 10px;
+    margin-bottom: 16px;
+  }
+  &__container {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+  &__pricetext {
+    line-height: 18px;
+    font-weight: 400;
+    color: $dark-color;
+  }
+  &__price {
+    line-height: 18px;
+    font-size: 16px;
+    font-weight: bold;
+    color: $title-color;
+  }
+  &__link {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 166px;
+    height: 40px;
+    font-size: 14px;
+    background-color: $button-color;
+    border-radius: 5px;
+    margin: 0 auto;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: $white-color;
+    border: 1px solid $button-color;
+    font-weight: 700;
+  }
+  &__link:hover {
+    color: $button-color;
+    background-color: $white-color;
+    transition: all ease-in 0.3s;
+  }
+  &__stars--block {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
```

Comment:
Cards and stars should be a different elements, don't mash them into one

Reply:
Oh, you should look to 66 line, sorry for that. You have cards__stars selector or smth like that. You need to style them only by card and stars, not mash them together